### PR TITLE
#2 Create project structure

### PR DESCRIPTION
**Pull request by petrokramar**
Connect to #2

### CODE REVIEW

File: src/main/java/juja/microservices/teams/exceptions/ApiErrorStatus.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package juja.microservices.teams.exceptions;
+
+public enum ApiErrorStatus {
+    TEAMS_EXCEPTION(
+            "0",
+            "Oops something went wrong :(",
+            "The exception is general in gamification"
```
Comment (OlgaKulikova) [inline src/main/java/juja/microservices/teams/exceptions/ApiErrorStatus.java]: Please change to the "The exception is general in team microservice"

**Comment by OlgaKulikova** [comment]
@starikivan Please look at the comment and add the endpoints' methods into the controller (just segnature of methods without implementation).

**Comment by petrokramar** [comment]
Signature of endpoints' methods added into the controller.

File: src/main/java/juja/microservices/teams/controller/TeamsController.java [hunk 2]
```diff
@@ -1,4 +1,50 @@
...
+    private TeamsService teamsService;
+
+    @PostMapping(value = "/teams", consumes = "application/json", produces = "application/json")
+    public ResponseEntity<?> addTeam(@Valid @RequestBody TeamRequest request) {
+        //TODO Should be implemented feature TMF-F1
+        return null;
+    }
+
+    @PutMapping(value = "/teams/{id}", consumes = "application/json", produces = "application/json")
+    public ResponseEntity<?> deactivateTeam(@PathVariable String id) {
```
Comment (OlgaKulikova) [inline src/main/java/juja/microservices/teams/controller/TeamsController.java]: According Technical Requirements this endpoind has the body {"from":"..."}

**Review by OlgaKulikova — CHANGES_REQUESTED** [review]

**Review by OlgaKulikova — APPROVED** [review]

**Pull request merged**